packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -0,0 +1,32 @@
+# WebDriverIO Testing
+
+The terra-functional-testing library is a utility for developing tests using the [WebdriverIO Framework](https://webdriver.io/).
+
+## Accessibility Testing
+
+The testing library integrates [axe-core](https://github.com/dequelabs/axe-core) utilities for automating accessibility testing.
+
+### Terra.validates.accessibility
+
+The accessibility assertion is available globally. Running the assertion will run acessibility checks on the current document. If acessibility violations are found the the test step will fail. 
```

Comment:
```suggestion
The accessibility assertion is available globally. Running the assertion will run acessibility checks on the current document. If acessibility violations are found, the the test step will fail. 
```

Reply:
Fixed typos and enabled my spell checker for .mdx files:

https://github.com/cerner/terra-toolkit/pull/452/commits/1aad2e2c77dc3a59fed32b3ce8c763a95b57d952